doc/source/cookbook.rst

```diff
@@ -5,24 +5,17 @@
 .. ipython:: python
    :suppress:
 
-   import pandas as pd
+   import datetime
+   import functools
+   import glob
+   import itertools
+   import os
    import numpy as np
```

Comment:
`os` import is out of order and could use a blank line before / after NumPy import (divide between standard library, third party and own code)

Reply:
I think `os` is in the right position, can you doble check?

Reply:
Momentarily thought `itertools` started with a "t". Please ignore